### DISCUSSION TITLE
Background color

### DISCUSSION THREAD

**Discussion by sfeast** [timbru31/cordova-plugin-lottie-splashscreen#414] [Q&A] [answered]
The color of the initial launch image can't be changed correct? e.g. it is always white until the app has taken focus & then it changes to what is specified for LottieBackgroundColor (at least on iOS). Or am I doing something wrong?

**Comment by timbru31** [comment, marked as answer]
Yes, especially for iOS you **must** configure the LaunchScreens (static images).

What we did in the past: The designers supplied the first frame of the animation as a static image we've used for the launch screen.
Therefore there was no visual gap between the launch of the app (tap by the user) and the moment some code has been loaded and Lottie kicks off.
